subsys/bluetooth/controller/ll_sw/ull_master.c

```diff
@@ -431,6 +431,8 @@ void ull_master_setup(memq_link_t *link, struct node_rx_hdr *rx,
 	struct pdu_adv *pdu_tx;
 	u32_t ticker_status;
 	u8_t chan_sel;
+	u8_t peer_addr[BDADDR_SIZE];
```

Comment:
Use reverse tree order, sorting the declarations.

Reply:
Sure, the won't be grouped together anymore though.